src/graphql/fragments/postFieldsFragment.js

```diff
@@ -125,7 +126,31 @@ const postFieldsFragment = withComments => `
     region
   }
   peopleReactedTotal
-  myVote
+  proposalStatus
+  proposalType
+  quorum
+  proposalOptions {
```

Comment:
so here is where if we are always loading this for every post the backend must only actual do this query for proposals

Reply:
It'll just return an empty set. Is there a bookshelf.js way to say "only look this up if the post is type === proposal"?

Reply:
yes, see my comment in hylo-node